Makefile

```diff
@@ -129,7 +129,7 @@ install:
 clean:
 	@rm -rf ./build ./vendor
 
-docker:
+docker-build:
 	docker build -t ${DOCKER_IMAGE}:${DOCKER_TAG} .
```

Comment:
i also changed this command because i was unable to run make docker. Changing the name allows it to run.